### PR TITLE
Migration changes for captcha

### PR DESCRIPTION
**Pull request by yashmsonkusare**

### CODE REVIEW

File: captcha-validation-service/pom.xml [hunk 1]
```diff
@@ -110,12 +127,12 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-ref-idobjectvalidator</artifactId>
-					<version>1.2.0-rc1</version>
+					<version>1.2.0.1-B2</version>
```
Comment (LoganathanSekar7627) [inline captcha-validation-service/pom.xml:130]: Point to migrated version. Migrate it if it is not yet migrated.

File: captcha-validation-service/pom.xml [hunk 2]
```diff
@@ -110,12 +127,12 @@
...
 				</dependency>
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-transliteration-icu4j</artifactId>
-					<version>1.2.0-rc1</version>
+					<version>1.2.0.1-B2</version>
```
Comment (LoganathanSekar7627) [inline captcha-validation-service/pom.xml:135]: Point to migrated version. Migrate it if it is not yet migrated.

File: pom.xml [hunk 3]
```diff
@@ -102,35 +102,35 @@
...
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
+<!--        <profile>-->
+<!--            <id>sonar</id>-->
+<!--            <properties>-->
+<!--                <sonar.sources>.</sonar.sources>-->
```
Comment (LoganathanSekar7627) [inline pom.xml:108]: Why is this commented? We need sonar enabled.

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

File: captcha-validation-service/pom.xml (hunk 1 above)
Comment (yashmsonkusare) [reply captcha-validation-service/pom.xml]: Done with the changes

File: captcha-validation-service/pom.xml (hunk 2 above)
Comment (yashmsonkusare) [reply captcha-validation-service/pom.xml]: Done with the changes

File: pom.xml (hunk 3 above)
Comment (yashmsonkusare) [reply pom.xml]: Enabled the sonar

**Review by LoganathanSekar7627 — APPROVED** [review]

**Review by ckm007 — COMMENTED** [review]
pom version can be 0.9.0-java21-SNAPSHOT

**Review by ckm007 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**